### DISCUSSION TITLE
Azure AD OIDC Error

### DISCUSSION THREAD

**Discussion by aaronbolton** [open-webui/open-webui#3856] [General]
When connected to Azure AD as an Enterprise App the following error is received.

{"detail":"The email or password provided is incorrect. Please check for typos and try logging in again."}

is there an option/variable I can enable on WebUI to debug the authentication process

**Comment by aaronbolton** [comment]
I resolved this issue it was just a mid-configuration by me but it would still be good to be able to debug this kind of issue as I would have be able to fix it sooner

**Reply by AlbertoManuel07** [reply]
How did you solve this problem? Could you share your experience with us?

**Reply by aaronbolton** [reply]
im the config I used Secret ID instead of Value from EntraID so just got them confused

**Reply by KingstonJoseph12** [reply]
I tried with providing both  secret ID and Value from Entra ID for MICROSOFT CLIENT SECRET. Both has failed.

**Comment by diwakar-s-maurya** [comment]
It can also happen if `WEBUI_SESSION_COOKIE_SAME_SITE` is set to `strict` and `WEBUI_SESSION_COOKIE_SECURE` is set to `True`.
`WEBUI_SESSION_COOKIE_SAME_SITE` should be set to `lax`.
